tests/unit/test_charm.py

```diff
@@ -885,3 +894,22 @@ def test_set_password_provided(self, connection):
 
         # verify app data is updated and results are reported to user
         self.assertEqual("canonical123", new_password)
+
+    @patch_network_get(private_address="1.1.1.1")
+    @patch("charm.MongoDBCharm.get_backup_service")
+    @patch("charm.MongoDBBackups._get_pbm_status")
+    def test_set_backup_password_pbm_busy(self, pbm_status, get_backup_service):
```

Comment:
**thought, out-of-scope** I was somewhat surprised to see that some tests are with unittest and some other with pytest. It would be good to unify those with the same framework, but this clearly out-of-scope in this PR, but possibly worth adding a ticket in Jira (not too much at the bottom of the backlog :D)

Reply:
I created two epics for refactoring and unifying tests in VM and K8S charm - there is a lot work to do there.